src/js/tech/tech.js

```diff
@@ -18,32 +18,35 @@ import document from 'global/document';
  */
 class Tech extends Component {
 
-  constructor(player, options={}, ready=function(){}){
+  constructor(options={}, ready=function(){}){
+    options = options || {};
     // we don't want the tech to report user activity automatically.
     // This is done manually in addControlsListeners
     options.reportTouchActivity = false;
-    super(player, options, ready);
+    super(null, options, ready);
 
     // Manually track progress in cases where the browser/flash player doesn't report it.
-    if (!this['featuresProgressEvents']) {
+    if (!this.featuresProgressEvents) {
       this.manualProgressOn();
     }
 
     // Manually track timeupdates in cases where the browser/flash player doesn't report it.
-    if (!this['featuresTimeupdateEvents']) {
+    if (!this.featuresTimeupdateEvents) {
       this.manualTimeUpdatesOn();
     }
 
     this.initControlsListeners();
 
-    if (options['nativeCaptions'] === false || options['nativeTextTracks'] === false) {
-      this['featuresNativeTextTracks'] = false;
+    if (options.nativeCaptions === false || options.nativeTextTracks === false) {
+      this.featuresNativeTextTracks = false;
     }
 
-    if (!this['featuresNativeTextTracks']) {
+    if (!this.featuresNativeTextTracks) {
       this.emulateTextTracks();
     }
 
+    this.textTracks_ = options.textTracks;
```

Comment:
I wonder if it might be better to merge these? Though, I guess we shouldn't ever have `textTracks_` already set in the constructor.